src/mod/module.h

```diff
@@ -501,6 +504,7 @@
 #define tcl_resultint ((int (*)(void))global[300])
 #define tcl_resultstring ((const char *(*)(void))global[301])
 #define getdccfamilyaddr ((int (*) (sockname_t *, char *, socklen_t, int))global[302])
+#define strncpyz (global[303])
```

Comment:
shouldn't we export strlcpy() instead and just define strncpyz for modules?

Reply:
i dont understand yet / i am unsure what you mean.

so, you want to replace all strncpy() with strlcpy() inside eggdrops core? i hesitated only because such a big patch wouldnt be easy to merge. but if you give me green lite, ill do a bigger patch here and export strncpyz() only for backward compatibility only to modules. if it is possible to only export to modules and not expose also to core. ill figure out then.